### ISSUE TITLE
ManageAudience.get_audience_groups requires all args and does not accept 'None' as a valid value.

### ISSUE THREAD

**Issue by MakotoAtsu** [line/line-bot-sdk-python#498]
## System Informations

* Python version: 3.11
* SDK version: 3.1.0
* OS:  M2 MAC OS

## Expected Behavior

The function [linebot.v3.audience.ManageAudience.get_audience_groups_with_http_info](https://github.com/line/line-bot-sdk-python/blob/7296de7119ecf61feb2299fcb08f7688488a0820/linebot/v3/audience/api/manage_audience.py#L1213C9-L1213C43)
 should send a Http Request when called with only the 'page' arg.

## Current Behavior

When I call the function with only the ```page``` arg, it throws a validation failure exception from Pydantic. 
According to the [official docs](https://developers.line.biz/en/reference/messaging-api/#get-audience-groups), the other args are optional. 
However, the function still forces Pydantic to validate all args (and it also doesn't accept None). 
We should either add a default value to all args or make it accept None.

Alternatively, if this function is generated from [line-openapi](https://github.com/line/line-openapi), 
I think we should set [required](https://github.com/line/line-openapi/blob/1cbcd909cd7c713cfd24a1ece5609dccac3ee0c1/manage-audience.yml#L403C1-L403C1) to false for non-mandatory arguments.

## Steps to Reproduce

Both
```py
try:
    with ApiClient(configuration=Configuration(access_token='xxx')) as api_client:
        api_instance = ManageAudience(api_client=api_client)
        api_response = api_instance.get_audience_groups_with_http_info(page=1)
        print(api_response)
except Exception as e:
    print("Exception when calling ManageAudience->get_audience_groups: %s\n" % e)
```
and
```py
try:
    with ApiClient(configuration=Configuration(access_token='xxx')) as api_client:
        api_instance = ManageAudience(api_client=api_client)
        api_response = api_instance.get_audience_groups_with_http_info(
            page=1,
            description=None,
            status=None,
            size=None,
            includes_external_public_groups=None,
            create_route=None)
            print(api_response)
except Exception as e:
    print("Exception when calling ManageAudience->get_audience_groups: %s\n" % e)
```
throw exception
```
print(e)
5 validation errors for GetAudienceGroupsWithHttpInfo
description
  field required (type=value_error.missing)
status
  field required (type=value_error.missing)
size
  field required (type=value_error.missing)
includes_external_public_groups
  field required (type=value_error.missing)
create_route
  field required (type=value_error.missing)
```
and 
```
print(e)
5 validation errors for GetAudienceGroupsWithHttpInfo
description
  none is not an allowed value (type=type_error.none.not_allowed)
status
  none is not an allowed value (type=type_error.none.not_allowed)
size
  none is not an allowed value (type=type_error.none.not_allowed)
includes_external_public_groups
  none is not an allowed value (type=type_error.none.not_allowed)
create_route
  none is not an allowed value (type=type_error.none.not_allowed)
```

**Comment by Yang-33** [comment]
Thank you for your report! As you wrote, there are difference between document and line-openapi. I'll communicate with internal teams to check which one is correct.

**Comment by Yang-33** [comment]
@MakotoAtsu As you said, line-openapi was wrong(https://github.com/line/line-openapi/pull/28). It's fixed in `3.2.0` and it's released. Could you check it please?

**Comment by MakotoAtsu** [comment]
Great!
I've already upgraded to v3.2.0 and it's working! 
I will close this issue. 

Thanks for your help.
